Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -156,6 +175,9 @@ - (void)updateTextFieldsAndCaptions {
             break;
         case STPAddressFieldTypeCountry:
             self.textField.keyboardType = UIKeyboardTypeDefault;
+            if (@available(iOS 10.0, *)) {
+                self.textField.textContentType = UITextContentTypeCountryName;
```

Comment:
does this autofill the full name of the country? if so, i'm not sure if autofill will correctly update `contents` (which should be a country code).

Reply:
Hmm. It does **not** provide quicktype suggestions (when tested on 11.2 and 11.0.1), and I think that's due to our use of the `UIPickerView` (which handles updating `contents`, etc).

This does seem potentially dangerous, because that *could* change in the future. I don't think this is adding anything, so I'm leaning toward removing it.

Reply:
Ah yeah, +1 for removing it then.